Phonebook.Backend/Phonebook.Backend.PictureService/Phonebook.Backend.PictureService/Startup.cs

```diff
@@ -0,0 +1,174 @@
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Startup"/> class.
+        /// </summary>
+        /// <param name="configuration"></param>
+        public Startup(IConfiguration configuration)
+        {
+            this.Configuration = configuration;
+            this.AppSettings = configuration.Get<PictureServiceConfiguration>();
+        }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Validation of the appsettings on Startup
+            services.AddSingleton<PictureServiceConfiguration>(AppSettings);
+            services.UseConfigurationValidation();
+            services.ConfigureValidatableSetting<PictureServiceConfiguration>(Configuration);
+            services.ConfigureValidatableSetting<ContactInformation>(Configuration.GetSection("ContactInformation"));
+            services.AddSingleton<PictureServiceConfiguration>(AppSettings);
+
+            // Add Api Versioning Logic
+            services.AddApiVersioning(o =>
+            {
+                o.ReportApiVersions = true;
+                o.AssumeDefaultVersionWhenUnspecified = true;
+                o.DefaultApiVersion = new ApiVersion(1, 0);
+
+            });
+
+            services.AddVersionedApiExplorer(options =>
+            {
+                // add the versioned api explorer, which also adds IApiVersionDescriptionProvider service
+                // note: the specified format code will format the version as "'v'major[.minor][-status]"
+                options.GroupNameFormat = "'v'VVV";
+                // note: this option is only necessary when versioning by url segment. the SubstitutionFormat
+                // can also be used to control the format of the API version in route templates
+                options.SubstituteApiVersionInUrl = true;
+            });
+
+            //Enable IIS Integration            
+            services.AddAuthentication(IISDefaults.AuthenticationScheme);
+            services.Configure<IISOptions>(options =>
+            {
+                options.AutomaticAuthentication = true;
+            });
+            services.AddCors(o => o.AddPolicy(CorsPolicy, builder =>
+            {
+                // Add some CORS allowed origins here
+                builder.WithOrigins(this.AppSettings.AllowedCORSDomains).AllowAnyMethod().AllowAnyHeader().AllowCredentials();
+            }));
+
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+
+
+
+            services.AddImageProcessingSettings(Configuration);
+
+            // Scheduled Tasks
+            services.AddSingleton<IScheduledTaskOptions<PurgeTask>>(
+                new ScheduledTaskOptions<PurgeTask>
+                {
+                    // Once every day at startup -> May be exectuted multiple times a day depending on recycling

```

Comment:
Can we add this comment to the `AppSettings.PurgeSchedule` and delete it here.

Reply:
Done.